core/trino-parser/src/main/java/io/trino/sql/parser/SqlParser.java

```diff
@@ -48,7 +49,7 @@
 
 public class SqlParser
 {
-    private static final BaseErrorListener LEXER_ERROR_LISTENER = new BaseErrorListener()
+    private static final ANTLRErrorListener LEXER_ERROR_LISTENER = new BaseErrorListener()
```

Comment:
Why?

Reply:
This is the interface needed by ANTLR. The base class is just a convenience for implementations. It’s like declaring as `Map` rather than `AbstractMap`.